src/components/views/settings/DevicesPanel.js

```diff
@@ -124,16 +124,21 @@ export default class DevicesPanel extends React.Component {
             // pop up an interactive auth dialog
             const InteractiveAuthDialog = sdk.getComponent("dialogs.InteractiveAuthDialog");
 
+            const singularDevice = this.state.selectedDevices.length === 1;
             const dialogAesthetics = {
                 [SSOAuthEntry.PHASE_PREAUTH]: {
                     title: _t("Use Single Sign On to continue"),
-                    body: _t("Confirm deleting these sessions by using Single Sign On to prove your identity."),
+                    body: singularDevice
```

Comment:
Can we please use the count feature of our internationalisation platform here in case some language has some funky pluralisation for 111 devices or whatever.

Reply:
Do we have docs on that? I've never had to use it as far as I'm aware.

Reply:
https://github.com/matrix-org/matrix-react-sdk/blob/faab35738f3686a609d6e0f2072f05e4a20ddc7f/src/i18n/strings/en_EN.json#L254-L255
https://github.com/matrix-org/matrix-react-sdk/blob/d470f9649d800f6c8f83c99b32fb602510981277/src/TextForEvent.js#L304

Don't know of any docs other than the https://github.com/martinandert/counterpart#pluralization

Reply:
Fixed. I've also filed https://github.com/vector-im/riot-web/issues/13060 as this is an opportunity for the design to be violated.